### ISSUE TITLE
improve column layout

### ISSUE THREAD

**Issue by grosser** [ddworken/hishtory#59]
atm there is so much dead space that should be used for displaying commands
![SCR-20221217-pfu](https://user-images.githubusercontent.com/11367/208278267-04b86f41-5f89-41f6-94bb-eac7334f5950.png)

- I don't need hostname
- cwd is less important than command -> go to back
- timestamp: use YYYY-MM-DD HH:MM:SS (zone is the local zone so no need to mention)
- Runtime -> time (or maybe ⌛)
-  Exit Code -> Exit

so in summary what I want is:
command, exit, cwd, at, time
(last 4 don't matter)
... maybe even change the hovered line to only display command so there is a way to see super-long commands (even going so far to make it take multiple lines if necessary ... maybe max 2-4)

**Comment by ddworken** [comment]
This is actually mostly already supported, you can customize the displayed columns and the order in which they are displayed. The timestamp format is also customizable. To do most of this:

```
hishtory config-set displayed-columns Command 'Exit Code' CWD Runtime Timestamp
hishtory config-set timestamp-format '2006-01-02 15:04:39'
```

Which then yields:

```
Search Query: > ls

┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Command                                                                                                                           Exit Code  CWD              Runtime       Timestamp            │
│──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────│
│ docker logs 91b785c29bd7 2>&1 | grep deep                                                                                         0          ~/               429ms         2022-12-18 01:04:19  │
│ docker logs 91b785c29bd7 | grep deep                                                                                              0          ~/               958ms         2022-12-18 01:04:19  │
│ docker logs 91b785c29bd7                                                                                                          0          ~/               1.28s         2022-12-18 01:04:19  │
│ docker ps                                                                                                                         0          ~/               1.042s        2022-12-17 23:21:119 │
...
```

I think the only remaining bit would be supporting customization of the column names.I filed #60 to track this.

**Comment by grosser** [comment]
awesome :D
maybe change `Custom Columns` -> `Custom Ctrl+R Columns` in the readme
